### ISSUE TITLE
Crash to Safe Mode

### ISSUE THREAD

**Issue by melphoi** [AnemoneTeam/Anemone#40]
Dear Dev..
This is my first time installing Anemone to replace the old Winterboard
I have some tweaks installed before Anemone, and after I installed Anemone, the device respringed to crash which lead to the Safe Mode
I already tried to uninstall most of the tweaks which left me the only Cydia and Icleaner but still it crashed into safe mode
Could you please help me to examine the crash log ?
Unfortunately I can't upload the notepad about the crash log due to the restriction but I already capture some parts of it.

Thanks in advance.

![untitled](https://cloud.githubusercontent.com/assets/15280037/10711539/c471e624-7aa8-11e5-96df-1f4fb9b44b8a.jpg)

**Comment by madroix** [comment]
+1 having similar issue

**Comment by coolstar** [comment]
Can you get a syslog?

**Comment by madroix** [comment]
I can if you would point me in the right direction to retrieve one.

**Comment by melphoi** [comment]
Dear Dev,

I already sent you both Crash and Sys Log from email.
Sorry for being late to introduce myself, I'm @thoteles on Twitter who'se asking for your help.

**Comment by melphoi** [comment]
Here's the link for [Syslog](http://www.writeurl.com/text/zb0bw5vvwq9edqz870zb/tcwgt7bd1sfgoirzk4z8/isr5ofdso7zig1stgslp) and [Crashlog](http://www.writeurl.com/text/68ycx40cc239vucyo0sw/tag7wqmlgrdfdr76z432/0sxh10i7jd8d08e0ozju)

**Comment by melphoi** [comment]
is there any solution or fix for this ?

**Comment by cj81499** [comment]
Are you using the latest beta?

**Comment by melphoi** [comment]
Yes.. 1.1-22 then 1.1-31 but both keep crashing and put my phone into safe mode

**Comment by cj81499** [comment]
Huh. Strange.

Try disabling all other tweaks with iCleaner Pro to see if it's a conflict with another tweak.

**Comment by melphoi** [comment]
I turned off all of Cydia Substrate Addons via Icleaner but not Substrate Safe Mode
Then I tried to install the latest 1.1-40 but it still crashed and again get into safe mode :(

**Comment by coolstar** [comment]
Can you try disabling all Anemone dylibs except for AnemoneCore and AnemoneUIKit and see if the issue is still there? If we can narrow it down to a specific part of Anemone it'll be easier to fix.

**Comment by melphoi** [comment]
It works !! at least for now it can get to Springboard without crashing even if I try to reboot the phone :D
Thanks,Dev.. :) 
what should I do next to make it fix without disabling most of the dylibs ?

Edit : According to your method, it seems Anemonedock.dylibs is the problem, i tried to turned on each dylibs one by one, and everything works, but when I turned on the dock one it crashed to safe mode, and I turned off the dock again then everything goes normal.

**Comment by coolstar** [comment]
ok, at least we're on to something now. Try re-enabling AnemoneDock and disabling themes one by on e until you find the culprit (target the dock themes first). It seems it might be an Info.plist that's corrupt.

**Comment by melphoi** [comment]
OK.. so based on your suggestion, here's what I do.. Correct me if I'm wrong..
Before I did this, I already have Classicdock tweak installed.
First, I turned off all of the Anemone's dylibs.. then :
Dock (ON) = crash to safe mode
Dock + Anemone.dylib = crash to safe mode
Dock + Anemone.dylib + Core = works
Dock + Anemone.dylib + Core + Icons = works
Dock + Anemone.dylib + Core + Icons + UIKit = works
Dock + Anemone.dylib + Core + Icons + UIKit + Colors = works
Dock + Anemone.dylib + Core + Icons + UIKit + Colors + Iconeffects = works

Well.. finally everyting is working :D
Kind of strage though, I didn't have any theme installed at the moment because I want to see that everything at least stable before I pick the theme.

**Comment by coolstar** [comment]
Yes, that is expected. All the other Anemone dylibs (including Anemone.dylib) depend on AnemoneCore.dylib.

**Comment by melphoi** [comment]
In the first place, it really confuse me like "what is wrong with my phone.." lol
None of the "safe mode fix tips" works with this Anemone problem on my phone until I came here and get your suggestion

I hope this "ON/OFF Cydia Substrate tips" could help people who might be has the same problem like me.
At least for now it's stable and I just tried installed some themes and no problem so far.
Great tweak, awesome Dev.

Thanks again,Dev Coolstar. :)
